docs/getting-started/create-configuration.md

```diff
@@ -631,7 +631,7 @@ you may specify a specific package by using the `-f` flag.
 ```console
 # Set this to the Docker Hub username or OCI registry you wish to use.
 REG=my-package-repo
-kubectl crossplane push configuration ${REG}/getting-started-with-aws-with-vpc:master
```

Comment:
were these ones still using `master` just missed in the 1.2 release process?

these should be `1.2.2`, not `1.1.2`, right?

Reply:
> were these ones still using master just missed in the 1.2 release process?

Yes 👍 

> these should be 1.2.2, not 1.1.2, right?

Good catch! Updating now :)